src/react/useQueries.ts

```diff
@@ -20,18 +22,30 @@ export function useQueries(queries: UseQueryOptions[]): UseQueryResult[] {
     return defaultedOptions
   })
 
-  const [observer] = React.useState(
-    () => new QueriesObserver(queryClient, defaultedQueries)
+  const observerRef = React.useRef(
+    new QueriesObserver(queryClient, defaultedQueries)
```

Comment:
why are we moving away from state to refs please? I explicitly made that change not too long ago :) It's the preferred way of doing one-time initializations -  I don't see us _writing_ to `observerRef.current` anyhwere?